src/client/functions.ts

```diff
@@ -7,17 +7,7 @@ type CallbackFn<T> = (data: T, cb: NuiCallbackFunc) => void;
  *  @param event - The event name to listen for
  *  @param callback - The callback function
  */
-export const RegisterNuiCB = <T = any>(event: string, callback: CallbackFn<T>): void => {
+export const RegisterNuiCB = <T = unknown>(event: string, callback: CallbackFn<T>): void => {
```

Comment:
> <T = unknown>  

Stricter, yes, but also less forgiving to beginners with no experience in using generics. Lets just keep it as any, and anyone who wants strict typing can just use the generic.

Reply:
Fixed in 8c93f02